### ISSUE TITLE
[Bug Report]: Worldedit causing "connection reset" error

### ISSUE THREAD

**Issue by Yurillith** [TeamAOF/All-of-Fabric-7#321]
### Modpack Version

2.5.2

### Describe your issue.

Me and my friend wanted to add WorldEdit to our AOF 7 private server to facilitate building.
However, for some unknown reason, upon adding WorldEdit, whenever we try to connect to the server we get a "connection reset" error.
I've tried talking to WorldEdit devs but they could not identify any reason as to why this should happen

### Latest Log

https://gist.github.com/Th3B4dD3m0n9118/be529c2ed94a4eab8737c4ce40bbc79a

### Have you modified the modpack?

Yes

### User Modifications

We have only added:
NETS (Not Enough Trinket Slots) version 0.3.5.5
Puzzles Lib version 8.1.25 - 1.20.1
Enchanting Infuser version 8.0.3 - 1.20.1
WorldEdit version 7.2.15

### Did the issue happen in singleplayer or on a server?

Server

### Discord Username

im4d3m0n_

**Comment by Yurillith** [comment]
Update:
Even without WorldEdit, after leaving the server for roughly an hour or even less than that, and trying to join back, you still get the same "connection reset" error, and the server console reports it had an "error receiving packet 23" just like when WorldEdit was applied

Also, client-side we have replaced embeddium with Sodium, Sodium Extra and Reese's Sodium Options, I forgot to mention that originally.

**Comment by Krutonium** [comment]
I'm seeing the same issue.

**Comment by SubordinalBlue** [comment]
Hmm... ~~not that'd I'd expect those changes to be at fault.~~

We've always had WorldEdit added to the official servers, no problems.


Er, I meant the embeddium/sodium swap; the other ones, idk.

**Comment by Krutonium** [comment]
I did more testing and it doesn't seem to be WorldEdit; Not sure what it is though.

**Comment by SubordinalBlue** [comment]
Is the problem there with _no_ mod changes/deletions/adds from base pack?

Could be; best to isolate is it really the changes, or a bug in the base pack.

**Comment by Krutonium** [comment]
> Is the problem there with _no_ mod changes/deletions/adds from base pack?
> 
> Could be; best to isolate is it really the changes, or a bug in the base pack.

Yes, I experienced it on a completely unmodified server. (but that's not what we're running atm)

**Comment by SubordinalBlue** [comment]
Unclear yet if your issue, Krutonium, should be in the same issue thread; maybe?

Waiting on OP to see if they have it with an unmodified server.

Either way, got logs?

**Comment by Krutonium** [comment]
Give me a minute to distill one from an unmodified client and server

**Comment by Krutonium** [comment]
![Image](https://github.com/user-attachments/assets/3f06218d-3f03-48f8-aef3-b70858f664f0)

[latest.log | Server](https://github.com/user-attachments/files/18612200/latest.log)

[latest.log | Client](https://github.com/user-attachments/files/18612204/latest.log)

This is one of the 2 errors I've seen, with the other one (that I can't make show up this time) being an unexpected packet 28

**Comment by SubordinalBlue** [comment]
Does that happen with a fresh player with no inventory/armor/trinkets/etc at all? (wild guess)


p.s. We're getting chatty here (which is fine)... maybe open an issue in #support on the discord?

**Comment by Krutonium** [comment]
Doesn't matter if the player is a new player or not

**Comment by SubordinalBlue** [comment]
Every login, or (annoyingly) just some?

**Comment by Krutonium** [comment]
Most, somtimes multiple times in a row, usually *at least* once.

**Comment by Yurillith** [comment]
> Unclear yet if your issue, Krutonium, should be in the same issue thread; maybe?
> 
> Waiting on OP to see if they have it with an unmodified server.
> 
> Either way, got logs?

Yes, we tested the mod with an unmodified version of the modpack on the server and it still led to the same error

Apologies for replying so late by the way, I had kind of forgotten about this bug report since it has been a while since I made it
